Update shared content topics (#5845)
grafana/agent: docs/sources/shared/flow/reference/components/otelcol-debug-metrics-block.md

The following arguments are supported:

Name                               | Type      | Description                                          | Default | Required
-----------------------------------|-----------|------------------------------------------------------|---------|---------
`disable_high_cardinality_metrics` | `boolean` | Whether to disable certain high cardinality metrics. | `false` | no

`disable_high_cardinality_metrics` is the Grafana Agent equivalent to the `telemetry.disableHighCardinalityMetrics` feature gate in the OpenTelemetry Collector.
It removes attributes that could cause high cardinality metrics.

> For example, attributes with IP addresses and port numbers in metrics about HTTP and gRPC connections will be removed.

Author: Yeah I waffled on that one. Looking at with fresh eyes this morning, I'll change it to `are removed` in the next commit.

> For example, attributes with IP addresses and port numbers in metrics about HTTP and gRPC connections will be removed.

Author: ```suggestion
For example, attributes with IP addresses and port numbers in metrics about HTTP and gRPC connections are removed.
```
